### ISSUE TITLE
Premium ddlto - Download don't start

### ISSUE THREAD

**Issue by ms235** [pyload/pyload#3862]
If got problems with starting a Download with premium ddlto Account.
I allready try to disable xFileSharing without success.

Here the log:

```
17.12.2020 10:17:53 | INFO | HOSTER XFileSharing[1172]: DdownloadCom \| Checking for link errors...
-- | -- | --
422 | 17.12.2020 10:17:53 | INFO | HOSTER XFileSharing[1172]: DdownloadCom \| No errors found
423 | 17.12.2020 10:17:53 | INFO | HOSTER XFileSharing[1172]: DdownloadCom \| Grabbing link info...
424 | 17.12.2020 10:17:53 | INFO | HOSTER XFileSharing[1172]: DdownloadCom \| Link name: 85yfjon50fvj
425 | 17.12.2020 10:17:53 | INFO | HOSTER XFileSharing[1172]: DdownloadCom \| Link size: N/D
426 | 17.12.2020 10:17:53 | INFO | HOSTER XFileSharing[1172]: DdownloadCom \| Link status: starte
427 | 17.12.2020 10:17:53 | INFO | HOSTER XFileSharing[1172]: DdownloadCom \| Processing as free download...
428 | 17.12.2020 10:17:53 | INFO | HOSTER XFileSharing[1172]: DdownloadCom \| Checking for link errors...
429 | 17.12.2020 10:17:53 | INFO | HOSTER XFileSharing[1172]: DdownloadCom \| No errors found
430 | 17.12.2020 10:17:53 | WARNING | CAPTCHA XFileSharing[1172]: DdownloadCom \| ReCaptcha \| Secure Token pattern not found
431 | 17.12.2020 10:17:53 | WARNING | CAPTCHA XFileSharing[1172]: DdownloadCom \| ReCaptcha \| Secure Token pattern not found
432 | 17.12.2020 10:17:53 | WARNING | CAPTCHA XFileSharing[1172]: DdownloadCom \| ReCaptcha \| reCAPTCHA noscript is blocked, trying reCAPTCHA interactive
433 | 17.12.2020 10:17:53 | WARNING | CAPTCHA XFileSharing[1173]: DdownloadCom \| ReCaptcha \| Invalid captcha  \| No captcha result obtained in appropriate timing (300s)
434 | 17.12.2020 10:17:53 | INFO | HOSTER XFileSharing[1173]: DdownloadCom \| Waiting 1 seconds...
435 | 17.12.2020 10:17:55 | INFO | Download restarted: ipzzxivm9zmc (1) \| Retry Captcha
436 | 17.12.2020 10:17:55 | INFO | Download starts: ipzzxivm9zmc (1)
437 | 17.12.2020 10:17:55 | WARNING | HOSTER XFileSharing[1173]: DdownloadCom \| Plugin may be unstable
438 | 17.12.2020 10:17:55 | INFO | HOSTER XFileSharing[1173]: DdownloadCom \| Grabbing link info...
439 | 17.12.2020 10:17:56 | INFO | HOSTER XFileSharing[1173]: DdownloadCom \| Link name: ipzzxivm9zmc
```

**Comment by ms235** [comment]
problem solved by an update of pyload from 0.4.9 to 0.4.20

**Comment by GammaC0de** [comment]
> problem solved by an update of pyload from 0.4.9 to 0.4.20

👍
